notebooks/01_structured.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<p>could you also add a docstring to this one. as a TODO we could add examples of the functions in this notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>done and TODO added to our project board</p>